contrib/ci/AzureCI/hypershift-install-azure.yaml

```diff
@@ -0,0 +1,131 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: hypershift-ops
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: hypershift-ops
+  name: admin
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: "hypershift-ops:admin"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    namespace: hypershift-ops
+    name: admin
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: hypershift
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: hypershift-operator
+  namespace: hypershift
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: quay.io/hypershift/hypershift-operator:latest
```

Comment:
Latest is coming out from Main, do we want that? (I'm just asking, I'n not fully sure if we want a more stable e.g 4.15 version or latest one)

Reply:
I think we do. Seth was working on updating e2e's to test with main since that's what we use in prod.

Reply:
Same was done for our AWS Root CI so I assume its okay.

Reply:
Cool then :)